src/backend/logconfig.py

```diff
@@ -22,6 +23,13 @@ def filter(self, record: logging.LogRecord) -> bool:
         return not record.name.startswith("watchfiles")
 
 
+def gzip_rotator(source, dest):
+    os.rename(source, dest)
+    with open(dest, 'rb') as f_in, gzip.open(f"{dest}.gz", 'wb') as f_out:
+        f_out.writelines(f_in)
```

Comment:
```suggestion
        f_out.write(f_in)
```
wieso writelines() und nicht write()?

Reply:
damit das ganze chunked gezipped wird. bei write müsste ich `.write(f_in.read())` schreiben und das wäre insgesammt ineffizienter